content/blog/pulumi-copilot-rest/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Announcing the Pulumi Copilot REST API Preview"
+date: 2024-12-12T15:15:42-05:00
+draft: false
+meta_desc: Explore Pulumi Copilot's new REST API preview for IaC.
+meta_image: meta.png
+authors:
+    - artur-laksberg
+tags:
+    - features
+    - ai
+    - llm
+    - copilot
+social:
+    twitter:
+    linkedin:
+
+# See the blogging docs at https://github.com/pulumi/docs/blob/master/BLOGGING.md
+# for details, and please remove these comments before submitting for review.
+---
+Since its initial release in June 2024, Pulumi Copilot has seen strong adoption, helping teams understand and manage their cloud infrastructure more effectively. Today, we're excited to announce the availability of the Pulumi Copilot REST API in preview. This release enables developers to integrate Pulumi Copilot's powerful capabilities directly into their own tools, applications, and workplace collaboration platforms. While currently in preview, we plan to quickly move the API to production as we gather feedback from early adopters and ensure it meets our community's needs.
```

Comment:
We should not talk about timeframes even in the abstract.  Let's just saw we want to gather feedback and requirements.